src/reporters/apiary-reporter.coffee

```diff
@@ -133,6 +137,19 @@ class ApiaryReporter
         logger.complete "See results in Apiary at: #{reportUrl}"
         callback()
 
+  _interestingAstToSend: (blueprintData = {}) ->
```

Comment:
Why `interesting`? Please name it properly. I can imagine something like `truncated` or `limited`.